File: MST/MCB.Business.Models/Trips/StopModel.cs
using MCB.Business.Models.Geo;
using MCB.Business.Models.WorldHeritages;
using System;

namespace MCB.Business.Models.Trips
{
    public class StopModel
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public int Order { get; set; }
        public DateTime Arrival { get; set; }
        public DateTime Departure { get; set; }
        public int TripId { get; set; }
        public double Latitude { get; set; }
        public double Longitude { get; set; }

        public CountryModel Country { get; set; } 
        public WorldHeritageModel WorldHeritage { get; set; }
    }
}

File: MST/MCB.AngularClient/MCBAngularClient/src/app/core/model/Stop/stop-model.ts
import { Country } from '../Geo/country.model';
import { WorldHeritage } from '../Geo/world-heritage.model';

export interface Stop {
  id: number;
  name: string;
  description: string;
  order: number;
  arrival: Date;
  departure: Date;
  tripId: number;
  latitude: number;
  longitude: number;
  country: Country;
  worldHeritage: WorldHeritage;
}
